include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -167,28 +167,36 @@ constexpr auto align_pairwise(sequence_t && sequences,
     // Configure the alignment algorithm.
     auto && [algorithm, complete_config] = detail::alignment_configurator::configure<decltype(seq_view)>(config);
 
-    using traits_t = detail::alignment_configuration_traits<remove_cvref_t<decltype(complete_config)>>;
-    //!brief Lambda function to translate specified parallel and or vectorised configurations into their execution rules.
-    constexpr auto get_execution_rule = [] ()
-    {
-        if constexpr (traits_t::is_parallel)
-            return seqan3::par;
-        else
-            return seqan3::seq;
```

Comment:
Are seqan3::par and seqan3::seq objects still used elsewhere or can they be deleted? It seems they have been replaced by detail::execution_handler_parallel and detail::execution_handler_sequential.

Reply:
No they are not needed anymore. They are publc API though. I would mark them deprecated instead. But in a separate PR. good point though.

Reply:
👍